GOVERNANCE.md

```diff
@@ -0,0 +1,54 @@
+# SlateDB Governance
+
+Anyone is welcome to contribute to SlateDB by [creating pull requests](CONTRIBUTING.md)
+or taking part in discussions about features, e.g. in [GitHub issues](https://github.com/slatedb/slatedb/issues).
+
+This document defines who has authority to do what on the SlateDB open-source project.
+
+While a formal assignment of authority is necessary as a fallback mechanism, it is
+not the way decisions regarding SlateDB are usually made, or expected to be made.
+The community sees disagreement as good as a diversity of perspectives having influence
+leads to better decisions. We seek to resolve disagreement by reaching 
+consensus based on the merit of technical arguments, with no regard to status or
+formal role. This almost always works.
+
+# SlateDB Roles
+
+There are two formal roles in SlateDB:
+
+* __SlateDB Committers__ have the right to merge pull requests into the SlateDB repo.
+* __The SlateDB Board__ has final authority over all decisions made on the project.
+
+# SlateDB Committers
+
+A __SlateDB Committer__ is able to merge pull requests into the SlateDB repositories under
+[https://github.com/slatedb](https://github.com/slatedb), including into
+the master branch that is released to production.
+
+To become a SlateDB Committer, start making PRs that are approved. When a sufficient
+trust is built through working on SlateDB this way, any current SlateDB Committer can nominate
+you to the board for the committer role.
+
+Every pull request should be approved by a SlateDB Committer other than the author of the
```

Comment:
I think we should have a higher standard for RFCs - maybe 2 committer approvals and no objecting votes from committers?